integration_tests/tests/query_tests/test_queries.py

```diff
@@ -490,6 +490,40 @@
                 ),
             },
         ),
+        (
+            "joined_spatial_aggregate",
+            {
+                "locations": {
+                    "query_kind": "daily_location",
+                    "date": "2016-01-01",
+                    "aggregation_unit": "admin3",
+                    "method": "last",
+                },
+                "metric": {
+                    "query_kind": "handset",
+                    "start_date": "2016-01-01",
+                    "end_date": "2016-01-02",
+                    "characteristic": "hnd_type",
+                    "method": "last",
+                },
+                "method": "dist",
+            },
+        ),
+        (
```

Comment:
If I am not mistaken this set of test parameters is an exact duplicate of the previous ones (just written in a different way), so it's sufficient to only keep one of them to avoid running the exact same test twice.